torchfix/torchfix.py

```diff
@@ -25,18 +25,55 @@
 
 DISABLED_BY_DEFAULT = ["TOR3", "TOR4"]
 
+ALL_VISITOR_CLS = [
+    TorchDeprecatedSymbolsVisitor,
+    TorchRequireGradVisitor,
+    TorchSynchronizedDataLoaderVisitor,
+    TorchVisionDeprecatedPretrainedVisitor,
+    TorchVisionDeprecatedToTensorVisitor,
+    TorchUnsafeLoadVisitor,
+    TorchReentrantCheckpointVisitor,
+]
+
+_ALL_ERROR_CODES = None
+
+def GET_ALL_ERROR_CODES():
+    global _ALL_ERROR_CODES
+    if _ALL_ERROR_CODES is None:
+        codes = []
+        for cls in ALL_VISITOR_CLS:
+            if isinstance(cls.ERROR_CODE, list):
+                codes += cls.ERROR_CODE
+            else:
+                codes.append(cls.ERROR_CODE)
+        _ALL_ERROR_CODES = codes
+    return _ALL_ERROR_CODES
 
 def GET_ALL_VISITORS():
-    return [
-        TorchDeprecatedSymbolsVisitor(DEPRECATED_CONFIG_PATH),
-        TorchRequireGradVisitor(),
-        TorchSynchronizedDataLoaderVisitor(),
-        TorchVisionDeprecatedPretrainedVisitor(),
-        TorchVisionDeprecatedToTensorVisitor(),
-        TorchUnsafeLoadVisitor(),
-        TorchReentrantCheckpointVisitor(),
-    ]
-
+    out = []
+    for v in ALL_VISITOR_CLS:
+        if v is TorchDeprecatedSymbolsVisitor:
+            out.append(v(DEPRECATED_CONFIG_PATH))
+        else:
+            out.append(v())
+    return out
+
+def get_visitor_with_error_code(error_code):
```

Comment:
The API (same for ruff and flake8) is that if, for example, `select=TOR0`, then TOR001, TOR002, TOR003 are enabled.
So we should check for `startswith`, not equality.
And multiple visitors can correspond to the same string in `select`.